crypto/dh/dh_lib.c

```diff
@@ -6,7 +6,7 @@
  * The implementation was written so as to conform with Netscapes SSL.
  *
  * This library is free for commercial and non-commercial use as long as
- * the following conditions are aheared to.  The following conditions
+ * the following conditions are ?aheared? to.  The following conditions
```

Comment:
This is the copyright text and CANNOT be changed anyway, as per Rich's comment below.